src/app/views/sidebar/sample-queries/SampleQueries.tsx

```diff
@@ -220,6 +230,10 @@ export class SampleQueries extends Component<ISampleQueriesProps, any> {
     const { tokenPresent } = this.props;
     const classes = classNames(this.props);
     let selectionDisabled = false;
+    const customStyles: Partial<IDetailsRowStyles> = {};
+    if (this.state.selectedQuery === props.item.id) {
+      customStyles.root = { backgroundColor: '#C0C0C0' };
```

Comment:
can this be put in a css or scss file?

Reply:
Thanks for pointing this out! I might have to check with the GE team because the other styling in this file is also in this format, not sure if I should be following their convention. Is it okay to leave it like this for now until then?

Reply:
yeah if theres other css in the file then its fine!